.github/workflows/fpga.yml

```diff
@@ -215,9 +302,9 @@ jobs:
 
       - name: Install kernel modules
         run: |
-          sudo apt update
-          sudo apt install -y make gcc
-          sudo ./hw/fpga/setup_fpga.sh
+          ls -l /tmp/caliptra-fpga-kmod
+          sudo insmod /tmp/caliptra-fpga-kmod/io_module.ko
+          sudo insmod /tmp/caliptra-fpga-kmod/rom_backdoor.ko
```

Comment:
nit: Is it woth making a separate script for 2.x and calling that here?

It's nice to have scripts in the repo to make it easy to run manually.

Reply:
The build script in`hw/fpga` covers this when running manually. The file paths involved are from the CI so this is not entirely re-usable locally.